portfolio/src/main/webapp/script.js

```diff
@@ -16,28 +16,70 @@ const VISIBLE = 'visible';
 const HIDDEN = 'hidden';
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random polaroid image to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['I\'m ready, I\'m ready, I\'m ready...', 
-      'You know what\'s funnier than 24? 25!', 
-      'East?! I thought you said Weast!', 
-      'Squidward, I used your clarinet to unclog my toilet!'];
-
-  // Picks a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Adds the random greeting to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function addRandomPolaroid() {
+  const images =
+      ['arches.JPG','cruise.jpg','dalat.jpg', 'mexico.jpg', 
+      'pittsburghSunset.jpg', 'saigonLightening.jpg', 
+      'saigonTower.jpg', 'thailand.jpg', 'yellowstone.JPG'];
+
+  // Picks a random image.
+  const image = images[Math.floor(Math.random() * images.length)];
+  
+  // Creates a div tag to store the polaroid image and description.
+  let div = document.createElement("div");
+  div.setAttribute('class', 'polaroid');
+
+  // Creates a p tag that contains the image name.
+  const imageName = image.split(".")[0];
+  let imageText = createPTag(imageName);
+
+  // Creates an img tag to store the image.
+  let imgTag = createImgTag(image);
+
+  div.appendChild(imgTag);
+  div.appendChild(imageText);
+  
+  // Adds the polaroid div tag to the page.
+  const imagesContainer = document.getElementById('images');
+  imagesContainer.appendChild(div);
```

Comment:
How can this be const if you append div?

Reply:
I believe declaring a variable as a constant just prevents reassignment. The object would still be mutable, which is how I was able to append to imagesContainer without a type error.

I updated it to be const anyways for safety.
Done

Reply:
Oh if you can keep it const, then do that. Also do div, that seems it can be const?

Reply:
done!